roles/database/tasks/main.yml

```diff
@@ -1,8 +1,12 @@
 ---
 - include: install-apt-deps.yml
-  when: initialize_local_db
+  when: fpsd_database_initialize_local_db
```

Comment:
Preserved this conditional during the reorg. Although the included task list installs postgres, it's being skipped because it's technically optional that the database is on the target host—it might instead be remote. (In which case, this becomes more of a database-client than a database-server role.) So, I think this conditional statement is fine as-is—but holler if you disagree, @fowlslegs.